src/main/java/com/github/jchartier/slackbot/pomodoro/dto/Pomodoro.java

```diff
@@ -0,0 +1,20 @@
+package com.github.jchartier.slackbot.pomodoro.dto;
```

Comment:
Is it really a DTO? Would it be more like a `model`... I don't know what is best